GlucoseOnFhir/BGMFhir.swift

```diff
@@ -0,0 +1,640 @@
+//
+//  BGMFhir.swift
+//  GlucoseOnFhir
+//
+//  Created by Kevin Tallevi on 7/11/17.
+//  Copyright © 2017 eHealth Innovation. All rights reserved.
+//
+
+// swiftlint:disable function_body_length
+// swiftlint:disable line_length
+// swiftlint:disable cyclomatic_complexity
+// swiftlint:disable file_length
+
+import Foundation
+import CCGlucose
+import SMART
+
+public class BGMFhir: NSObject {
+    static let BGMFhirInstance: BGMFhir = BGMFhir()
+    var patient: Patient?
+    var device: Device?
+    var deviceComponent: DeviceComponent?
+    var specimen: Specimen?
+    
+    var givenName: FHIRString = "Lisa"
+    var familyName: FHIRString = "Simpson"
```

Comment:
This doesn't look like a test class. is it? Fake data should be limited to a Debug or Test class.